### ISSUE TITLE
easygui_qt.demos "Error importing PyQt5" 

### ISSUE THREAD

**Issue by melellington** [aroberge/easygui_qt#34]
All buttons in the demo give me the error: 

Error importing PyQt5.  Have you installed it?
could not find function get_int

I am running Python version 3.9.0

imac:~ mel$  python -V
Python 3.9.0

![Screen Shot](https://user-images.githubusercontent.com/80646841/115720860-707c4880-a34b-11eb-9484-a2541c707f56.png)

**Comment by aroberge** [comment]
Look at the bottom of your screenshot: it shows that you are running Python version 2.7.

Note that I have not been working on this project for a few years now and I have very little free time to investigate problems.

**Comment by melellington** [comment]
If you look at the top of my screenshot
My terminal command "python -V" 
shows I am running version 3.9.0.

Your demo code has a bug in it 
which forces it to run version 2.7.  
overriding my default version 3.9.0.  

Perhaps your code cannot handle newer
versions of Python such as 3.9.0?

—Mel

**Comment by aroberge** [comment]
As I wrote in https://github.com/aroberge/easygui_qt/blob/master/setup.py, it was intended to run with Python 3, not Python 2. I did almost all my testing exclusively with Python 3.

The code on https://github.com/aroberge/easygui_qt/blob/master/easygui_qt/demos/launcher.py#L28 launches the default Python interpreter: it certainly does not force it to run a different version.

I don't have a mac but based on what I have read elsewhere, I suspect that, if you replace `python` by `python3` on that line, the demo will work correctly on your computer.
